### DISCUSSION TITLE
How to suppress repeated request for permissions?

### DISCUSSION THREAD

**Discussion by bizzbyster** [vpnhood/VpnHood#97] [Q&A] [answered]
Every time I start VpnHood client on my Windows laptop, I get that User Account Control message that asks "Do you want to allow this app to make changes to your device?" and the app is listed as ".NET Host". Is there some way to make a configuration change to suppress these?

**Comment by trudyhood** [comment, marked as answer]
VpnHood client on Windows needs to run as administrator to let WinDivert capture packets. .NET core runtime runs VpnHood that's why it needs the permission. We can bypass it by running a Windows Service. I have implemented it before in another project just need much more time :)
https://github.com/vpnhood/VpnHood/issues/35
